mlrun/runtimes/nuclio/application/application.py

```diff
@@ -172,7 +172,7 @@ def __init__(
 class ApplicationRuntime(RemoteRuntime):
     kind = "application"
 
-    @min_nuclio_versions("1.12.7")
+    @min_nuclio_versions("1.13.1")
```

Comment:
it worth changing `min_nuclio_versions` implementation that when using it on __init__ it will log the Class name (ApplicationRuntime) and not __init__ as it says nothing.
when using specific function (.deploy) it will log Class.deploy functionality not supported.